src/util/search-providers/imdb.ts

```diff
@@ -12,19 +12,26 @@ const IMDB_SEARCH_PARAMS = {
  * @param params - search parameters
  */
 export async function imdb(params: ISearchParameters): Promise<ISearchResult[]> {
-  let result;
 
   try {
-    result = await imdbAPI.search({ name: params.title, year: params.year }, IMDB_SEARCH_PARAMS);
+    const imdbID = /([tt])\w+/;
+    if (imdbID.exec(params.title)) {
+      let findById = await imdbAPI.get({ id: params.title }, IMDB_SEARCH_PARAMS);
+      return [
+        { id: findById.imdbid, title: findById.title, year: findById.year, posterUrl: findById.poster }
```

Comment:
You can notice that in [movie-search.ts](https://github.com/dmbaranov/evemovies-bot/blob/master/src/util/movie-search.ts#L41) there's filtering happens based on a movie year. It's done to avoid results pollution - the user wants to see only those movies that were released recently and are not available for downloading yet. But when the user searches by id only, I think it makes sense to return this movie no matter what year it was released. So my suggestion is to add a parameter like `avoid_filtering` and if it's passed, the movie shouldn't be filtered out.